public/app/features/provisioning/SyncRepository.tsx

```diff
@@ -40,7 +40,7 @@ export function SyncRepository({ repository }: Props) {
     if (!name) {
       return;
     }
-    syncResource({ name, body: {} }); // will queue a full resync job
+    syncResource({ name, body: { incremental: true } }); // will queue a full resync job
```

Comment:
when requesting from the UI -- do we want incremental?

I have been hitting the button when things are not correct -- so seems like we want a full sync job.

Incremental is good for background task and quick response to commit hooks.

Alternatively (but a bit weird!) this could be based on how long ago we ran a job -- if you ask to sync when things just ran... likely want a full sync